x/gov/handler_test.go

```diff
@@ -281,26 +293,104 @@ func TestCreateTextProposal(t *testing.T) {
 				},
 				RawOption: textOption,
 			},
+			Signers:        []weave.Condition{hAliceCond, hBobbyCond},
 			WantCheckErr:   errors.ErrInput,
 			WantDeliverErr: errors.ErrInput,
 		},
+		"A proposal creation is restricted to electorate members only": {
+			Init: func(t *testing.T, db weave.KVStore) {
+				createElectorate(t, db, []weave.Address{
+					hAlice,
+				})
+				createElectorate(t, db, []weave.Address{
+					hBobby,
+					hCharlie,
+				})
+			},
+			Msg: CreateProposalMsg{
+				Metadata: &weave.Metadata{Schema: 1},
+				Base: &CreateProposalMsgBase{
+					Title:          "my proposal",
+					Description:    "my description",
+					StartTime:      now.Add(time.Hour),
+					ElectionRuleID: weavetest.SequenceID(1),
+					Author:         hBobby,
+				},
+				RawOption: textOption,
+			},
+			// Both signers are from the second electorate while
+			// the proposal is created for the first one.
+			Signers:        []weave.Condition{hBobbyCond, hCharlieCond},
+			WantCheckErr:   errors.ErrUnauthorized,
+			WantDeliverErr: errors.ErrUnauthorized,
+		},
+		"A proposal creation can be signed by any electorate member": {
+			Init: func(t *testing.T, db weave.KVStore) {
+				createElectorate(t, db, []weave.Address{
+					hAlice,
+					hCharlie,
+				})
+				createElectorate(t, db, []weave.Address{
+					hBobby,
+					hCharlie,
+				})
+			},
+			Msg: CreateProposalMsg{
+				Metadata: &weave.Metadata{Schema: 1},
+				Base: &CreateProposalMsgBase{
+					Title:          "my proposal",
+					Description:    "my description",
+					StartTime:      now.Add(time.Hour),
+					ElectionRuleID: weavetest.SequenceID(1),
+					Author:         hBobby,
```

Comment:
This is an interesting scenario: the author of this proposal is not part of the electorate and not able to vote. It still makes sense for me as hBobby signed it, too who is a member. 
We could also restrict this to electorate members for simplicity.... Marking this for @kmw101  for the docs